common/jmeter/setup/TestData_SCIM2_Add_Tenant_User.jmx

```diff
@@ -0,0 +1,491 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Shall we rename the file to ...Add_tenant_User**_s_**?

Reply:
There is already a https://github.com/wso2/performance-is/blob/master/common/jmeter/setup/TestData_Add_Tenant_Users.jmx but it is not seems to be using SCIM

Reply:
Just add s at the end. TestData_SCIM2_Add_Tenant_User**s**.jmx

Reply:
Done